src/models/pieChart.js

```diff
@@ -208,8 +211,8 @@ nv.models.pieChart = function() {
         dispatch.tooltipShow(e);
     });
 
-    dispatch.on('tooltipShow', function(e) {
-        if (tooltips) showTooltip(e);
+    pie.dispatch.on('elementMouseout.tooltip', function(e) {
+        dispatch.tooltipHide(e);
     });
 
     dispatch.on('tooltipHide', function() {
```

Comment:
Hmm, should probably fix that everywhere then, but since it's the same in lineChart that woudl be out of scope for this commit so don't worry about that.